### PR TITLE
Fix multiline text alignment for MorePublicationRow

### PR DESCRIPTION
**Pull request by dishanest**
## Overview

Fix multiline text alignment for `MorePublicationRow`

## Screenshots

<details>
<summary> Before </summary>
<br>

IMG_1432

</details>

<details>
<summary> After </summary>
<br>

IMG_1433

</details>

### CODE REVIEW

File: Volume/View Models/ArticleInfo.swift [hunk 1]
```diff
@@ -25,11 +25,11 @@ struct ArticleInfo: View {
                 }
 
                 Text(article.title)
-                    .multilineTextAlignment(.leading)
                     .font(largeFont ? .latoBold(size: 24) : .latoBold(size: 16))
                     .lineLimit(3)
                     .padding(.top, 0.5)
                     .blur(radius: article.isNsfw ? 3 : 0)
```
Comment (Npikielny) [inline Volume/View Models/ArticleInfo.swift:31]: NSFW?!

**Review by Npikielny — APPROVED** [review]
Looks good! Some of the new lines in the project seem inconsistent, but nothing directly from this PR so ¯\_(ツ)_/¯

File: Volume/View Models/ArticleInfo.swift (hunk 1 above)
Comment (dishanest) [reply Volume/View Models/ArticleInfo.swift:31]: ( ͡° ͜ʖ ͡°)
<details>
<summary> Blurred NSFW article </summary>
<br>

IMG_1434

</details>

<details>
<summary> Article title </summary>
<br>

IMG_1435

</details>

**Review by NgaiJustin — APPROVED** [review]
looks good 👍

**Review by amyqxhuang — APPROVED** [review]
looks good! :)

**Pull request merged**